pkg/security/admission/admission.go

```diff
@@ -312,19 +284,17 @@ func (c *constraint) getNamespace(name string, ns *kapi.Namespace) (*kapi.Namesp
 
 // getMatchingSecurityContextConstraints returns constraints from the store that match the group,
 // uid, or user of the service account.
-func getMatchingSecurityContextConstraints(store cache.Store, userInfo user.Info) ([]*kapi.SecurityContextConstraints, error) {
+func GetMatchingSecurityContextConstraints(cache *securitycache.SecurityCache, userInfo user.Info) ([]*kapi.SecurityContextConstraints, error) {
```

Comment:
I think this function/refactor is actually worth promoting to an interface that finds matching SCCs.

```go
type SCCMatcher interface{
    FindApplicableSCCs(user user.Info)
}

type DefaultSCCMatcher struct{
    cache client.SCCLister
}
```